nsfailover.sh

```diff
@@ -140,17 +140,20 @@ info "Best nameserver is ${use_level} (${use_server})"
 #nameserver ${NS_1}
 #nameserver ${NS_2}
 #nameserver ${NS_3}
+
 resolvconf="${resolvconf}
 options timeout:${NS_TIMEOUT} attempts:${NS_ATTEMPTS}"
 # Optionally add search parameter
 [ -n "${NS_SEARCH}" ] && resolvconf="${resolvconf}
 search ${NS_SEARCH}"
 
 # Load current config (without comments)
-current="$(cat "${NS_FILE}" |egrep -v '^#')" || true
+current="$(cat "${NS_FILE}" | egrep -v '^#')" || true
+
+LessCommentResolv=$(echo "$resolvconf" | egrep -v '^#')
```

Comment:
```suggestion
resolvNoComment=$(echo "$resolvconf" | egrep -v '^#')
```

Reply:
I create a new commit to resolv this question.